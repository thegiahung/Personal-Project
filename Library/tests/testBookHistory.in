add member Python
add collection books.csv
rent 100000 111111
relinquish 100000 111111
book history 111111
exit
