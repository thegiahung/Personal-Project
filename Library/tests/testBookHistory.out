user: Success.

user: 17 books successfully added.

user: Success.

user: Success.

user: 100000

user: Ending Library process.
